src/Aspire.Dashboard/README.md

```diff
@@ -0,0 +1,11 @@
+# .NET Aspire Dashboard
+
+## As a dotnet tool
+
+The dashboard can be packaged and executed as a `dotnet tool`.
+
+Configuration is (currently) performed via environment variables:
+
+- `DOTNET_DASHBOARD_OTLP_ENDPOINT_URL` specifies the OTLP endpoint, and defaults to http://localhost:18889.
+- `ASPNETCORE_URLS` specifies the HTTP endpoint via which the dashboard web application is served, and defaults to http://localhost:18888.
+- `DOTNET_DASHBOARD_GRPC_ENDPOINT_URL` specifies the gRPC endpoint to which the dashboard connects to for its data, and defaults to http://localhost:18999.
```

Comment:
Is `DOTNET_DASHBOARD_GRPC_ENDPOINT_URL` the right name? When I read it, I think that it is a grpc endpoint on the dashboard. For example, `DOTNET_DASHBOARD_OTLP_ENDPOINT_URL` is the dashboard's OTLP endpoint address.

One of these isn't right.

Reply:
Good point. Perhaps `DOTNET_RESOURCE_SERVICE_ENDPOINT_URL` is a better name.

Reply:
It also no longer has a default. I'll update that too.

Reply:
> It also no longer has a default. I'll update that too.

Actually, the default is removed in https://github.com/dotnet/aspire/pull/1795, so I'll update the doc in that PR instead.